<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceStatusWidgetClass</class>
 <widget class="QWidget" name="DeviceStatusWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>136</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DeviceStatusWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">background:#444444;</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="20,149">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="20,50,50,50,880">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="buttonDeviceInfo">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">border:0 solid grey;border-top-right-radius:0px;
border-top-left-radius:8px;
border-bottom-right-radius:0px;
border-bottom-left-radius:8px;
border-right:0px;
color:#FFFFFF;
font:16px PingFang SC;</string>
          </property>
          <property name="text">
           <string>设备信息</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonDeviceConfig">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">border:0 solid grey;border-top-right-radius:0px;
border-top-left-radius:8px;
border-bottom-right-radius:0px;
border-bottom-left-radius:8px;
border-right:0px;
color:#FFFFFF;
font:16px PingFang SC;</string>
          </property>
          <property name="text">
           <string>设备配置</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonDeviceLog">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">border:0 solid grey;border-top-right-radius:0px;
border-top-left-radius:8px;
border-bottom-right-radius:0px;
border-bottom-left-radius:8px;
border-right:0px;
color:#FFFFFF;
font:16px PingFang SC;</string>
          </property>
          <property name="text">
           <string>设备日志</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>14</width>
            <height>13</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
